transforms/.make.transforms

```diff
@@ -166,6 +166,10 @@ extra-help:
 	@# Help: Test an already-built image (use make image) to be sure the --help option works
 	$(DOCKER) run -t --rm $(DOCKER_LOCAL_IMAGE) python $(TRANSFORM_RUNTIME_SRC_FILE) --help
 
+.transforms.test-spark_image-help::
+	@# Help: Test an already-built image (use make image) to be sure the --help option works
+	$(DOCKER) run -t --rm $(DOCKER_LOCAL_IMAGE) python3 $(TRANSFORM_RUNTIME_SRC_FILE) --help
```

Comment:
It seems you added this to use `python3` instead of `python`.   I guess `python` is not available inside the image?  Can we add link - we're building the image after all?

Reply:
Its more Spark convention. Afraid to break it, but will try